### PR TITLE
Fix from param

### PR DESCRIPTION
**Pull request by Joozty**
Fix for https://kmi-ou.atlassian.net/browse/CORE-3054

### CODE REVIEW

File: store/helpers/pages/page.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Page {
   }
 
   get isLast() {
-    return this.data.length < this.maxSize
+    return this.data.length === 0
```
Comment (viktor-yakubiv) [inline store/helpers/pages/page.js]: How is this affected?

**Comment by mcancellieri** [comment]
It's a bug in production, is fine to wait as our production is not really production yet :)

File: store/helpers/pages/page.js (hunk 1 above)
Comment (Joozty) [reply store/helpers/pages/page.js]: You can find more in the task description. Basically when 100 records are requested API can return only 10 but it doesn't mean there are no more records. The last page is detected when empty array is sent.

Comment (viktor-yakubiv) [reply store/helpers/pages/page.js]: Yes but this is not correct behaviour, is it? @valerapro could you please look at this?

Comment (Joozty) [reply store/helpers/pages/page.js]: It was fixed like that. I just did the frontend part.  Like I don't like how it works but I understand it's hard to process it on the backend. So two lines of changes on the frontend are sufficient for me.

**Pull request merged**